rust/platform/triple_mappings.bzl

```diff
@@ -186,9 +186,15 @@ def triple_to_constraint_set(triple):
         list: A list of constraints (each represented by a list of strings)
     """
     if triple == "wasm32-wasi":
-        return ["@io_bazel_rules_rust//rust/platform/cpu:wasm32", "@io_bazel_rules_rust//rust/platform/os:wasi"]
+        return [
+            "@rules_rust//rust/platform/cpu:wasm32",
```

Comment:
These strings get rendered into the BUILD files for the generated toolchains. There needs to be a way to detect the current `rules_rust` workspace name and render that into there. There may also be some potential for `repo_mapping` but that may also require the ability to dynamically detect the workspace name or force users to explicitly pass this data which to me seems undesirable.